cv32/sim/uvmt_cv32/xrun.mk

```diff
@@ -76,12 +76,20 @@ comp: mk_xrun_dir $(CV32E40P_PKG) $(OVP_MODEL_DPI)
 		-elaborate
 #		$(XRUN_VELABCOVERAGE)
 
-XRUN_COMP_RUN = $(XRUN_COMP_ARGS) $(XRUN_RUN_BASE_FLAGS)
+XRUN_COMP_RUN = $(XRUN_COMP) $(XRUN_RUN_BASE_FLAGS)
```

Comment:
I believe this line is redundant since XRUN_COMP_RUN is assigned to either way in the next if/else block.

Reply:
I updated a change. Removing the redundant assignment doesn't work (I don't have an explanation) so I simply have it default to multi step and then overwrite if single step.

Reply:
The mysteries of makefiles, ok no problem.  This can be merged.